discovery-infra/test_infra/helper_classes/kube_helpers/agent.py

```diff
@@ -114,3 +114,57 @@ def _attempt_to_get_status() -> dict:
     def approve(self) -> None:
         self.patch(approved=True)
         logger.info("approved agent %s", self.ref)
+
+    @staticmethod
+    def wait_for_agents_to_install(agents, nodes_number, timeout: Union[int, float] = DEFAULT_WAIT_FOR_CRD_STATUS_TIMEOUT) -> None:
+        Agent.wait_till_all_agents_are_in_status(
+            agents=agents,
+            nodes_count=nodes_number,
+            status=consts.AgentStatus.VALIDATED,
+            timeout=timeout,
+        )
+        Agent.wait_till_all_agents_are_in_status(
+            agents=agents,
+            nodes_count=nodes_number,
+            status=consts.AgentStatus.READY_FOR_INSTALLATION,
+            timeout=timeout,
+        )
+        Agent.wait_till_all_agents_are_in_status(
+            agents=agents,
+            nodes_count=nodes_number,
+            status=consts.AgentStatus.INSTALLED,
+            timeout=timeout,
+        )
+
+    @staticmethod
+    def are_agents_in_status(
+                    agents, nodes_count, status,
```

Comment:
Add typehints to all the functions

Reply:
Already added. https://github.com/openshift/assisted-test-infra/pull/895/files#diff-89bdb261814ae6f7398131ddf383103d5b27daa611d12d4cccc0160cb0c772e3R142 Thank you.

Reply:
I'm referring to the parameters